### PR TITLE
Android support improvements

### PR DESCRIPTION
**Pull request by xymus**

### CODE REVIEW

**Comment by Morriar** [comment]
+1

File: src/platform/android.nit [hunk 1]
```diff
@@ -278,6 +281,7 @@ $(call import-module,android/native_app_glue)
 	redef fun compile_c_code(compiler, compile_dir)
 	do
 		var android_project_root = android_project_root.as(not null)
+		var short_project_name = compiler.mainmodule.name.replace("-", "_")
```
Comment (privat) [inline src/platform/android.nit]: remonter `sort_project_name` en attribut lazy de la classe?

Comment (xymus) [inline src/platform/android.nit]: C'est dépendant de `compiler.mainmodule`, je pourrais probablement ajouter le compilateur comme attribut du toolchain..?

**Comment by xymus** [comment]
Rerolled to remove commits on sensors and hopefully pass testing. It really does need the full `mnit`, event after the 2nd try to PR!

**Comment by privat** [comment]
+1

**Pull request merged**